subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,190 @@
-Dependencies that had to undergo conflict resolution render the requested and selected version separated by a right arrow character.
-
 [[sec:identifying_reason_dependency_selection]]
-== Identifying which dependency version was selected and why
-
-Large software projects inevitably deal with an increased number of dependencies either through direct or transitive dependencies.
-The <<#sec:listing_dependencies,dependencies report>> provides you with the raw list of dependencies but does not explain _why_ they have been selected or _which_ dependency is responsible for pulling them into the graph.
+== Identify the Dependency Version Selected
 
-Let's have a look at a concrete example.
-A project may request two different versions of the same dependency either as direct or transitive dependency.
+A project may request two different versions of the same dependency either directly or transitively.
 Gradle applies <<dependency_resolution.adoc#sec:version-conflict,version conflict resolution>> to ensure that only one version of the dependency exists in the dependency graph.
-In this example the conflicting dependency is represented by `commons-codec:commons-codec`.
+The following example introduces a conflict with `commons-codec:commons-codec`, added both as a direct dependency and a transitive dependency of JGit:
 
-.Declaring the JGit dependency and a conflicting dependency
 ====
 include::sample[dir="snippets/dependencyManagement/inspectingDependencies-dependencyInsightReport/groovy",files="build.gradle[tags=dependency-declaration]"]
 include::sample[dir="snippets/dependencyManagement/inspectingDependencies-dependencyInsightReport/kotlin",files="build.gradle.kts[tags=dependency-declaration]"]
 ====
 
-The dependency tree in a link:https://scans.gradle.com/[build scan] renders the selection reason (conflict resolution) as well as the origin of a dependency if you click on a dependency and select the "Required By" tab.
+The dependency tree in a link:https://scans.gradle.com/[build scan] shows information about conflicts.
+Click on a dependency and select the "Required By" tab to see the selection reason and origin of the dependency.
 
-.Dependency insight capabilities in a build scan
 image::dependency-management-dependency-insight-report-build-scan.png[]
 
-Every Gradle project provides the task `dependencyInsight` to render the so-called  _dependency insight report_ from the command line.
-Given a dependency in the dependency graph you can identify the selection reason and track down the origin of the dependency selection.
-You can think of the dependency insight report as the inverse representation of the dependency report for a given dependency.
+=== Dependency Insights
 
-The task takes the following parameters:
+Gradle provides the built-in `dependencyInsight` task to render a  _dependency insight report_ from the command line.
+Dependency insights only provide information about a single project at a time.
```

Comment:
> ... only provide information about a single configuration and single project ...

Let's state both elements, since we covered them for `dependencies`